dataset/clip_generator.py

```diff
@@ -338,12 +341,16 @@ def clip_generator(
         subs = pysrt.open(script_path)
         text = " ".join([sub.text for sub in subs])
     else:
-        with open(script_path, "r", encoding=CHARACTER_ENCODING) as script_file:
+        with open(script_path, "r", encoding=constants.CHARACTER_ENCODING) as script_file:
             text = script_file.read()
 
     text = text.lower().strip().replace("\n", " ").replace("  ", " ")
+    symbols=get_symbols(transcription_model.language)
     invalid_chars = get_invalid_characters(text, symbols)
-    assert not invalid_chars, f"Invalid characters in text (missing from language): {','.join(invalid_chars)}"
+    detailed_info=""
+    for c in invalid_chars:
+        detailed_info+=f"{c} ({unicodedata.name(c)}),"
```

Comment:
You can simplify this to `",".join([f"{c} ({unicodedata.name(c)})" for c in invalid_chars])`